app/src/main/kotlin/cz/covid19cz/erouska/ui/sandbox/SandboxFragment.kt

```diff
@@ -3,28 +3,38 @@ package cz.covid19cz.erouska.ui.sandbox
 import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
+import android.view.View
 import cz.covid19cz.erouska.R
 import cz.covid19cz.erouska.databinding.FragmentSandboxBinding
 import cz.covid19cz.erouska.ui.base.BaseFragment
 import cz.covid19cz.erouska.ui.dashboard.event.GmsApiErrorEvent
 import cz.covid19cz.erouska.ui.sandbox.event.SnackbarEvent
+import kotlinx.android.synthetic.main.fragment_sandbox.*
 
 class SandboxFragment :
     BaseFragment<FragmentSandboxBinding, SandboxVM>(R.layout.fragment_sandbox, SandboxVM::class) {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        subscribe(GmsApiErrorEvent::class){
-            startIntentSenderForResult(it.status.resolution?.intentSender,
-                REQUEST_GMS_ERROR_RESOLUTION, null, 0, 0, 0, null)
+        subscribe(GmsApiErrorEvent::class) {
+//            startIntentSenderForResult(it.status.resolution?.intentSender,
```

Comment:
Proč je tohle zakomentované?

Reply:
Testoval jsem na non-whitelist účtu a způsobovalo mi to crash. Dnes budu starý mobil wipovat a připojím účet

Reply:
Komentář odeberu samozřejmě